README.md

```diff
@@ -169,134 +137,116 @@ Some changes do not require an RFC:
 If you're not sure whether your change requires an RFC, feel free to create an
 issue and ask.
 
-## What the process is
+## RFC Process
 
-In short, to get a major feature added to CDK/JSII, one usually first gets the
-RFC merged into the RFC repo as a markdown file. At that point the RFC is
-'ready' and may be implemented with the goal of eventual inclusion into
+In short, to get a major feature added to CDK/JSII, one usually writes an RFC 
+as a markdown file and gets it approved and merged into the RFC repo. At that point the RFC is
+'approved' and may be implemented with the goal of eventual inclusion into
 CDK/JSII.
```

Comment:
```suggestion
'approved' and may be implemented into CDK/jsii.
```

The "implemented with the goal of eventual inclusion" feels a bit "squishy" to me; if it's implemented and meets the design spec of the RFC, we'll work with the implementor to integrate it. Or is the intent to capture that there may be lag between "contributor implements" and "reviewed, pushed, and live in all published repos"?